src/cmd/linuxkit/docker/cmd.go

```diff
@@ -0,0 +1,103 @@
+package docker
+
+import (
+	"context"
+	"errors"
+	"io"
+	"os"
+
+	"github.com/containerd/containerd/reference"
+	dockertypes "github.com/docker/docker/api/types"
+	"github.com/docker/docker/api/types/container"
+	"github.com/docker/docker/client"
+	log "github.com/sirupsen/logrus"
+)
+
+// Client get a docker client.
+func Client() (*client.Client, error) {
+	// for maximum compatibility as we use nothing new
+	// 1.30 corresponds to Docker 17.06, supported until 2020.
+	err := os.Setenv("DOCKER_API_VERSION", "1.30")
```

Comment:
Maybe we should increment this a bit? 2020 is kind of ending...

Reply:
What version should be here? I am happy to change it to whatever you think is right.

Reply:
Let's leave this for now. If we want to change it, it should get its own PR so that anything that breaks is isolated.